spyder/widgets/editor.py

```diff
@@ -359,9 +360,21 @@ def __init__(self, parent, actions):
         close_all_but_this = create_action(self, _("Close all but this"),
                                            triggered=self.close_all_but_this)
 
-        self.menu_actions = actions + [None, fileswitcher_action,
+        actions = actions + [None, fileswitcher_action,
                                        symbolfinder_action,
-                                       copy_to_cb_action, None, close_right,
+                                       copy_to_cb_action]

```

Comment:
Please leave this as it was, and instead move the creation of `external_fileexp_action` before this.